.github/workflows/ruby.yml

```diff
@@ -68,4 +68,6 @@ jobs:
           bundle install --jobs 4
       - run: bin/setup ci
       - name: run spec
-        run: bin/rails spec
\ No newline at end of file
+        run: bin/rails spec
+      - name: rubocop
```

Comment:
Please reorder these. It's better to have rubocop fail first because it's quicker.